extension/chrome/elements/compose-modules/compose-input-module.ts

```diff
@@ -245,13 +245,13 @@ export class ComposeInputModule extends ViewModule<ComposeView> {
       /* eslint-disable @typescript-eslint/no-non-null-assertion */
       switch (recipient.sendingType) {
         case 'to':
-          result.to!.push({ email: recipient.email, name: recipient.name });
+          result.to!.push({ email: recipient.email, name: Xss.stripEmojis(recipient.name || '') });
```

Comment:
let's move recipient object creation outside of `switch` statement, so we won't need to duplicate code for `to`, `cc` and `bcc`:
```ts
const recipientObj = { email: recipient.email, name: Xss.stripEmojis(recipient.name || '') };
switch (recipient.sendingType) {
    case 'to':
        result.to!.push(recipientObj);
        break;
    case 'cc':
...
```

Reply:
thank you - will do.